### ISSUE TITLE
Issue with Slot

### ISSUE THREAD

**Issue by krab7191** [Hejtmus/svelte-lightbox#40]
When using default "something copyable" code [here](https://www.npmjs.com/package/svelte-lightbox#user-content-something-copyable) (only the Sapper import line removed), seeing this error: 
```
ValidationError: $$slots is an illegal variable name
node_modules/svelte-lightbox/src/Lightbox.svelte
93:       {:else if gallery}
94:         <InternalGallery {imagePreset}>
95:           {#if $$slots.thumbnail}
                   ^
96:             <div>
97:               <slot name="thumbnail"/>
ValidationError: $$slots is an illegal variable name
```

Svelte version 3, svelte-lightbox version 0.6.1

**Comment by Hejtmus** [comment]
# Hello,

thanks for your report. It looks like you are using version of Svelte that doesn't support `$$slots`. `$$slots` were added in `svelte 3.25.0`, please make sure you are using version of Svelte that supports `$$slots`.

I also tried to reproduce your issue, but it seems everything runs fine with latest Svelte.

Best regards,
Filip.

**Comment by Hejtmus** [comment]
I set peer dependency svelte to version 3.25.0 to prevent this kind of errors. Try upgrading svelte-lightbox to 0.6.2.

**Comment by Hejtmus** [comment]
Hi @krab7191, could you please give me feedback on bug-fix I released in version 0.6.2, so I can close this issue?

**Comment by Hejtmus** [comment]
No response, closing.
